handler.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"context"
+
+	"github.com/jackc/pgx/v4"
+)
+
+
+
+func Home(w http.ResponseWriter, r *http.Request){
+	fmt.Println("hit home page")
+}
+
+func GetLatestPostings(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+	conn, _ := pgx.Connect(ctx, "postgresql://admin:quest@localhost:8812/qdb")
+	defer conn.Close(ctx)
+
+	// Read top 10 latest posts
+	rows, _ := conn.Query(ctx, "SELECT id, price FROM postings4 order by price desc limit 10")
```

Comment:
Minor: use a query builder (overkill probably in this case) or read it directly from a .sql file in a resource directory. This way you can test/tune/iterate over your SQL query without having to recompile.